block.go

```diff
@@ -1243,11 +1265,18 @@ gatherlines:
 
 		// calculate the indentation
 		indent := 0
-		for indent < 4 && line+indent < i && data[line+indent] == ' ' {
-			indent++
+		indentIndex := 0
+		if data[line] == '\t' {
+			indentIndex++
+			indent += 4
+		} else {
+			for indent < 4 && line+indent < i && data[line+indent] == ' ' {
+				indent++
+				indentIndex++
+			}
 		}
```

Comment:
The spec says "List markers must be followed by one or more spaces *or* a tab" (emphasis mine), so I unified into the `else` clause in both places. Thanks!